packages/mui-material-nextjs/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
+    "skipLibCheck": true,
```

Comment:
What was failing here?

Reply:
Mismatched NextJS version. I just pinned the nextjs version and removed this compiler option.